app/presenters/tree_builder.rb

```diff
@@ -272,7 +272,7 @@ def add_to_sandbox
   def add_root_node(nodes)
     root = nodes.first
     root[:title], root[:tooltip], icon = root_options
-    root[:icon] = ActionController::Base.helpers.image_path("100/#{icon || 'folder'}.png")
+    root[:icon] = ActionController::Base.helpers.image_path("100/#{icon.try(:to_s) || 'folder'}.png")
```

Comment:
Since this is in double quotes, I believe it calls to_s under the covers.  So, I do not think you need to change this line.